hott.opam

```
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: [ "Jason Gross <jgross@mit.edu>" "Ali Caglayan <alizter@gmail.com>" ]
+homepage: "http://homotopytypetheory.org/"
+bug-reports: "https://github.com/HoTT/HoTT/issues"
+license: "BSD-2-Clause"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [make "install"]
+depends: [
+  "coq" {>= "8.13.2"}
```

This seems suspicious.  Are you sure it's not 
```suggestion
  "coq" {>= "8.13~"}
```
?

---

Yes, that seems to be more sensible. I didn't know about the tilde syntax, but checking the opam documentation it makes more sense to use.